frontend/packages/kubevirt-plugin/integration-tests-cypress/cypress.json

```diff
@@ -3,6 +3,8 @@
   "screenshotsFolder": "../../../gui_test_screenshots/cypress/screenshots",
   "videosFolder": "../../../gui_test_screenshots/cypress/videos",
   "video": true,
+  "videoCompression": false,
+  "experimentalRunEvents": true,
```

Comment:
Are you sure we still need this property to use the `after:spec` listener? According to https://github.com/cypress-io/cypress/issues/2522#issuecomment-827607381 it's no longer necessary.

Reply:
Remove it got an error like below:
```
 Error: The `after:spec` event requires the experimentalRunEvents flag to be enabled.

To enable it, set `"experimentalRunEvents": true` in your cypress.json
    at createErrorResult (/home/gouyang/.cache/Cypress/6.2.0/Cypress/resources/app/packages/server/lib/plugins/child/validate_event.js:3:71)
    ```